SL_Back:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Trajectory Controllers ---------------------------------------
  joint_trajectory_controller:
    type: position_controllers/JointTrajectoryController
    joints:
      - SL_Back_Left
      - SL_Back_Right

#    gains: # Required because we're controlling an effort interface
#      SL_Back_Left: {p: 50000,  d: 200, i: 100, i_clamp: 1}
#      SL_Back_Right: {p: 50000,  d: 200, i: 100, i_clamp: 1}

